code/index.html

```diff
@@ -1,22 +1,75 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
-    <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
-  </head>
-  <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width,initial-scale=1.0">
+  <title>Your Life Science News</title>
+  <link rel="stylesheet" href="./style.css" />
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Montserrat&family=Roboto:wght@100;300;400&display=swap" rel="stylesheet">
+  <!-- dont forget the meta tag for responsiveness -->
+</head>
+
+<body>
+  <div class="container">
+    <header class="header">
+      <div class="newssite-name">
+        <img class="logo" src="https://img.icons8.com/ios/50/000000/dna-helix.png" alt="logo" />
```

Comment:
A suggestion is to divide the image tag into smaller pieces example: 
<img 
class="logo"
scr="https://img.icon8.com/ios/50/000000/dna-helix.png"
alt=""logo"/>